docs/designs/sanitize-html.md

```diff
@@ -26,10 +27,11 @@ The sanitization is performed against user HTML, that is in most cases HTML tags
 
 ### Sanitization Modes
 
-There are 3 sanitization modes:
+There are 4 sanitization modes:
 
 - `standard` (*default*): Described below
 - `strict`: Disallow html
+- `strict-inline`: Based on strict, permitting only phrasing content. Block elements such as `blockquote`, `ul`, and `p` are not permitted.
 - `loose`: This is based on the `standard` mode, with a few more allowed HTML attributes like `class`, `style`. The final list came from the telemetry of the old hub/landing pages and archived contents.
```

Comment:
@jdanyow I do also wonder if a no-headings and no-extensions option would be great. It would support all basic markdown elements except headings.

Reply:
Is this something between `contentType: markdown` and `contentType: inlineMarkdown`? Like a markdown section on some startup pages that isn't strictly an inline markdown, but isn't the entire page either? If almost feels like another type of markdown, following the current convention that could look something like `contentType: markdownSection`, then we could explorer what restrictions are imposed on markdown sections, like disallow h1 and such.